docs/CircleCI.md

```diff
@@ -0,0 +1,66 @@
+# CircleCI
+
+The Buildkite Migration tool's currently supported, partially supported and unsupported properties in translation CircleCI pipelines to Buildkite pipelines are listed below:
+
+## Supported properties
+
+### aliases
+
+An `alias` is a definition of a reusable snippet of a CircleCI pipeline. Defined with a `&` (anchor) - these are then substituted into the required section of the pipeline with `*`: for example, `*tests`.
+
+The Buildkite Migration tool supports reading configuration defined from anchors and utilised through the use of an `alias`.
+
+### commands
+
+A `command` defined in a CircleCI pipeline is a reusable set of instructions with parameters that can be inserted into required `job` executions. Commands can have their own set list of `steps` that are translated through to the generated [command step](https://buildkite.com/docs/pipelines/command-step)'s `commands`. If a `command` contains a `parameters` key, its default parameters that are specified within the block is passed through into the specific command if it is utilised.
```

Comment:
I think that the parameters part is written in a confusing way. Maybe something like:

```suggestion
A `command` defined in a CircleCI pipeline is a reusable set of instructions with parameters that can be inserted into required `job` executions. Commands can have their own set list of `steps` that are translated through to the generated [command step](https://buildkite.com/docs/pipelines/command-step)'s `commands`. If a `command` contains a `parameters` key, they are respected when used in jobs/workflows and their defaults values used when not specified.
```

Reply:
Reworded - I think that sounds better!